app/constants.py

```diff
@@ -268,7 +268,16 @@ class OrganisationUserPermissionTypes(enum.StrEnum):
 # Service callbacks
 DELIVERY_STATUS_CALLBACK_TYPE = "delivery_status"
 COMPLAINT_CALLBACK_TYPE = "complaint"
-SERVICE_CALLBACK_TYPES = [DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE]
+RETURNED_LETTERS_CALLBACK = "returned_letters"
+SERVICE_CALLBACK_TYPES = [DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE, RETURNED_LETTERS_CALLBACK]
+
+
+class ServiceCallbackTypes(enum.StrEnum):
+    delivery_status = "delivery_status"
+    complaint = "complaint"
+    returned_letters = "returned_letters"
+    inbound_sms = "inbound_sms"
```

Comment:
I feel like it’s confusing to add this now. It means anyone looking at the code needs to understand that this is not used. Why not wait until we are going to use it to add it?

Reply:
I see your point on this and will remove the enum from this PR.